gis/rsgislib/Portfile

```diff
@@ -1,110 +1,152 @@
-}
-
-patchfiles          patch-CMakeLists.txt.diff \
-                    patch-RSGISHistoCubeFileIO_cpp.diff
-
-extract.mkdir       yes
-
-# /bin/sh: line 0: cd: build/python: No such file or directory
-cmake.out_of_source no
-
-# want -std=c++11 otherwise doesn’t compile with GDAL 2.3+
-compiler.cxx_standard       2011
+compiler.cxx_standard       2014
 compiler.blacklist-append   {clang < 340}
-configure.cxxflags-append   -std=c++11
 
-configure.args-append   -DCGAL_INCLUDE_DIR=${prefix}/include \
-                        -DCGAL_LIB_PATH=${prefix}/lib \
-                        -DFFTW_INCLUDE_DIR=${prefix}/include \
-                        -DFFTW_LIB_PATH=${prefix}/lib \
-                        -DGDAL_INCLUDE_DIR=${prefix}/include \
-                        -DGDAL_LIB_PATH=${prefix}/lib \
-                        -DGEOS_INCLUDE_DIR=${prefix}/include \
-                        -DGEOS_LIB_PATH=${prefix}/lib \
-                        -DGMP_INCLUDE_DIR=${prefix}/include \
-                        -DGMP_LIB_PATH=${prefix}/lib \
+configure.args-append   -DBOOST_INCLUDE_DIR=[boost::include_dir] \
+                        -DBOOST_LIB_PATH=[boost::lib_dir] \
                         -DGSL_INCLUDE_DIR=${prefix}/include \
                         -DGSL_LIB_PATH=${prefix}/lib \
-                        -DHDF5_INCLUDE_DIR=${prefix}/include \
-                        -DHDF5_LIB_PATH=${prefix}/lib \
-                        -DINSTALL_PYTHON_USING_PREFIX=ON \
-                        -DKEA_INCLUDE_DIR=${prefix}/include \
-                        -DKEA_LIB_PATH=${prefix}/lib \
-                        -DMPFR_INCLUDE_DIR=${prefix}/include \
-                        -DMPFR_LIB_PATH=${prefix}/lib \
+                        -DGDAL_INCLUDE_DIR=${prefix}/include \
+                        -DGDAL_LIB_PATH=${prefix}/lib \
                         -DMUPARSER_INCLUDE_DIR=${prefix}/include \
                         -DMUPARSER_LIB_PATH=${prefix}/lib \
-                        -DXERCESC_INCLUDE_DIR=${prefix}/include \
-                        -DXERCESC_LIB_PATH=${prefix}/lib
+                        -DKEA_INCLUDE_DIR=${prefix}/include \
+                        -DKEA_LIB_PATH=${prefix}/lib
+
+configure.args-append   -DRSGISLIB_WITH_UTILTIES=ON \
+                        -DRSGISLIB_WITH_DATA=OFF \
+                        -DRSGIS_PYTHON=OFF
 
-variant python27 description {Build with Python 2.7 bindings} \
-                 conflicts python35 python36 python37 {
 
-    depends_build-append    port:py27-numpy
-    configure.args-append   -DPYTHON_EXE=${prefix}/bin/python2.7
-    configure.args-append   -DPYTHON_INSTALL_PREFIX=${destroot}${frameworks_dir}/Python.framework/Versions/2.7/
+# Figure out HDF5's mpi include directory
+set mpl_include_dir ""
+if {![catch {set result [active_variants hdf5 openmpi]}]} {
+    if {$result} {
+        set mpl_include_dir "-I${prefix}/include/openmpi-mp"
+    }
+}
+if {![catch {set result [active_variants hdf5 mpi]}]} {
```

Comment:
Are you checking whether `hdf5` is built with `mpich`?

If so, the correct variant name is `mpich`.

Reply:
> Are you checking whether `hdf5` is built with `mpich`?

Yes, indeed.

> If so, the correct variant name is `mpich`.

Pushed correction.

(`gdal` Portfile needs a re-work!)